### PR TITLE
Add BSC to VRF v2 configurations

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

**Review by that1guy — COMMENTED** [review]
I can confirm all the keyhashes and coordinator/token addresses are correct in this documentation.  I'll wait for @pramodhs2 to confirm the LINK premium amount as I am unsure.

File: docs/Using Randomness/vrf-contracts.md [hunk 1]
```diff
@@ -46,9 +48,6 @@ Fee parameters are configured in the coordinator contract and specify the premiu
 |500 gwei Key Hash|`0xff8dedfbfa60af186cf3c830acbc32c05aae823045ae5ea7da1e45fbfaba4f92`|
 |1000 gwei Key Hash|`0x9fe0eebf5e446e3c998ec9bb19951541aee00bb90ea201ae456421a2ded86805`|
 |Premium|0.25 LINK|
-|Minimum Confirmations|3|
-|Maximum Confirmations|200|
-|Maximum Random Values|500|
```
Comment (pramodhs2) [inline docs/Using Randomness/vrf-contracts.md:51]: why are we removing the confirmations and max random values?

Comment (pramodhs2) [reply docs/Using Randomness/vrf-contracts.md:51]: Rest of the information looks good 👍

Comment (dwightjl) [reply docs/Using Randomness/vrf-contracts.md]: They're the same for all networks right now, but I'll restore them.

**Review by pramodhs2 — APPROVED** [review]

**Pull request merged**